portal/lib/paylib.php

```diff
@@ -37,7 +37,7 @@
 use OpenEMR\Common\Crypto\CryptoGen;
 
 if ($_SESSION['portal_init'] !== true) {
-    $_SESSION['whereto'] = '#paymentcard';
+    OpenEMR\Common\Session\SessionUtil::setSession('whereto', '#paymentcard');
```

Comment:
This is another place where will be creating a Openemr core session inside the portal with this function. This script works in both openemr and portal and performance not an issue. so would directly set the session variables (ie. go back to line that was removed) and then use flag prior to the globals.php include that turns off the session lock write so direct setting of session will also work when this script is used in openemr core.